File: f1-api/controllers/mock/circuit.go
package mock

import "f1-api/model"

// CircuitController :
type CircuitController struct{}

// GetCircuits :
func (cc CircuitController) GetCircuits() ([]model.Circuit, error) {

	circuits := []model.Circuit{
		model.Circuit{
			Name:     "Circuit1",
			Country:  "Country1",
			Distance: 1902,
			Turns:    5,
		},
		model.Circuit{
			Name:     "Circuit2",
			Country:  "Country2",
			Distance: 1222,
			Turns:    1,
		},
	}

	return circuits, nil
}

File: f1-discord-bot/model/circuit.js
class Circuit{

    constructor(name = '', distance = -1, country = '', turns = -1){
        
        this.name = name
        this.distance = distance
        this.country = country
        this.turns = turns
    }

    static fromJSON(jsonCircuit){
        
        let name = ''
        let distance = ''
        let country = ''
        let turns = ''

        if (jsonCircuit.hasOwnProperty('Name')){
            name = jsonCircuit['Name']
        }
        if (jsonCircuit.hasOwnProperty('Distance')){
            distance = jsonCircuit['Distance']
        }
        if (jsonCircuit.hasOwnProperty('Country')){
            country = jsonCircuit['Country']
        }
        if (jsonCircuit.hasOwnProperty('Turns')){
            turns = jsonCircuit['Turns']
        }

        return new Circuit(name = name, distance = distance, country = country, turns = turns)
    }
}

module.exports = {
    Circuit : Circuit
}